c2corg_ui/templates/utils/__init__.py

```diff
@@ -82,3 +82,30 @@ def has_associations(doc):
         a.get('all_routes') or a.get('routes') or \
         a.get('recent_outings') or a.get('outings') or \
         a.get('articles') or a.get('xreports') or a.get('books')
+
+
+def get_route_gear_articles(route):
+    articles = []
+    if route.get('activities'):
+        if ('snowshoeing' in route.get('activities') or
```

Comment:
I tried to follow the recommendations here: https://www.python.org/dev/peps/pep-0008/
I'll adapt the formatting.